### ISSUE TITLE
Label picker only submits label changes to GitHub only after undo window passes

### ISSUE THREAD

**Issue by damithc** [HubTurbo/HubTurbo#923]
Noticed some unusual behavior when applying labels.

1. Applied label.
2. The labels showed up on bView almost immediately, although the undo window is still open.
3. The labels did not show up on the pView until the next sync.

May be a periodic sync happened just as I was applying the labels?

**Comment by damithc** [comment]
I tried a few times. Seems the label shows up on bView almost immediately while the action is still undoable.

**Comment by lejolly** [comment]
yup, the changes are sent immediately, if the user decides to undo them, then the original labels are sent again.

**Comment by damithc** [comment]
That's not my intention. We should hold back the change to give the user a chance to undo the change. The whole idea is to avoid making user mistakes public. Just like Gmail doesn't actually send the email during the undo window. Do as a separate issue?

**Comment by lejolly** [comment]
Sure, or just rename this issue?
